src/index.ts

```diff
@@ -142,30 +120,42 @@ export default <ExportedHandler<Env>>{
 											},
 										});
 									}
+									if (!qMessage.ok) {
```

Comment:
Not too much of a concern if this errors imo, it'll just get reset on the next question (we could add a / command to re-send if needed?)

Reply:
I removed the returned error, but I retained the check(inverted), so that it only stores the ID in the DO if it is actually successful in sending the message.